adding truncate to KB article and rewrite intro sentence (#1524)
temporalio/documentation: kb/temporal-cluster-install-options.md

---
slug: install-temporal-cluster-for-development
title: All the ways to run a Temporal Cluster for development
tags:
  - kb-article
date: 2022-06-07T00:00:00Z
---

> There are many ways to run a [Temporal Cluster](/concepts/what-is-a-temporal-cluster) for development and testing purposes.
> This article aims to maintain a comprehensive of list them.

Reviewer: ```suggestion
You can run a [Temporal Cluster](/concepts/what-is-a-temporal-cluster) for development and testing purposes in various ways.
This article aims to maintain a comprehensive list of them.
```
